core/schema/container.graphqls

```diff
@@ -297,6 +297,9 @@ type Container {
     """
     address: String!
   ): Container!
+
+  "The unique image reference"
+  sha: String
```

Comment:
Same here, I would just call it `imageRef`.

Reply:
fixed